miden/tests/integration/cli/cli_test.rs

```diff
@@ -38,3 +40,40 @@ fn cli_run() -> Result<(), Box<dyn std::error::Error>> {
 
     Ok(())
 }
+
+#[test]
+fn cli_bundle_debug() {
+    let mut cmd = bin_under_test().command();
+    cmd.arg("bundle").arg("--debug").arg("./masm-examples/bundle/lib");
+    cmd.assert().success();
```

Comment:
As discussed offline, are we able to check the `MastForest.decorators` field for some use of `AsmOp` here?

Reply:
I gave it a try by testing if there are any decorators at all. Is that the right way?